payload/src/main/java/io/jenkins/jenkinsfile/runner/Runner.java

```diff
@@ -113,4 +137,13 @@ private void writeLogTo(PrintStream out) throws IOException, InterruptedExceptio
             }
         }
     }
+
+    private void addCredentials(Credentials creds) throws IOException {
```

Comment:
This and `getStore()` made sense as separate functions when we were explicitly loading credentials in a loop (or particularly in this case, during the brief moment I was supporting both credentials files and embedded credentials in the scm file) but I feel like keeping them as separate methods still makes sense in case we want to do things like that in the future.